libs/shared-lib/src/lib/Card/CommonCard.tsx

```
@@ -9,7 +9,21 @@ import Typography from '@mui/material/Typography';
+interface ContentItem {
+  name: string;
+  gradeLevel: string[];
+  language: string[];
+  artifactUrl: string;
+  identifier: string;
+  appIcon: string;
+  contentType: string;
+  mimeType: string;
+  description: string;
+  posterImage: string;
+  children: [{}];
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)